ts4.1/index.d.ts

```diff
@@ -184,15 +191,15 @@ type NormalizeWithKeyPrefix<
     : Normalize<T[K]>
   : never;
 
-type KeyPrefix<N extends Namespace> =
+export type KeyPrefix<N extends Namespace> =
```

Comment:
Same above, we only want to expose types that can be used externally

Reply:
Same answer as above :)